daft/runners/pyrunner.py

```diff
@@ -337,20 +337,27 @@ def run_iter_tables(
 
     @contextlib.contextmanager
     def actor_pool_context(
-        self, name: str, resource_request: ResourceRequest, num_actors: int, projection: ExpressionsProjection
+        self,
+        name: str,
+        actor_resource_request: ResourceRequest,
+        _: ResourceRequest,
```

Comment:
What is this, why do we need it?

If this is the task-based one we can also do `_task_resource_request`